docs/API.md

```diff
@@ -464,6 +466,16 @@ An object with the following properties:
 > TODO: Clarify the default value of each optional `"params"` property.
 >
 
+*priotity*
+
+This parameter is a free integer that will be used by the zonemaster testing backend daemon to sort the test requests from highest to lowest priority.
+This parameter will typically be used in a setup where a GUI will send requests to the bachend and would like to get response as soon as possible while at the same time using the idle time for background bach testing.
```

Comment:
"the zonemaster testing backend daemon" --> The Zonemaster Backend *Worker* daemon"

What is a "free integer"?